cockatrice/src/dlg_settings.cpp

```diff
@@ -438,6 +442,7 @@ void UserInterfaceSettingsPage::retranslateUi()
     annotateTokensCheckBox.setText(tr("Annotate card text on tokens"));
     animationGroupBox->setTitle(tr("Animation settings"));
     tapAnimationCheckBox.setText(tr("&Tap/untap animation"));
+    idleClientTimeOutCheckBox.setText(tr("Disconnect from server if sitting idle for an extended periods of time"));
```

Comment:
`an extended periods` is not gramatically correct.

Let's go with `Disconnect from server if idle for (the exact time limit)`